specifications/base.protocol.md

```diff
@@ -0,0 +1,106 @@
+# Base Concepts
+
+This section defines the core identity concepts used by the current specification.
+
+## Decentralization
+
+Decentralization is achieved in the following ways:
+
+- **Self-Issued Identity Tokens** - Each participant is responsible for creating and cryptographically signing its own
+  identity tokens. A central identity provider is therefore not required, eliminating a potential source of network
+  failure.
+- **Participant Verifiable Credential and Cryptographic Material Management** - Each participant is responsible for
+  storing Verifiable Credentials and presenting them as Verifiable Presentations (VPs). Each participant is also
+  responsible for storing and making its own cryptographic material available to Verifiers. This specification defines
+  how VCs are presented and verified without the need for a third-party verification service.
```

Comment:
We could file this as a topic to be discussed. I think it is still valid though because it uses the term "verification service". What is needed is the public key material and the status list VC, which can be cached by a verifier, hence no need for a service. 

We could also clarify this further.